src/wxProfilerGUI/threadsview.cpp

```diff
@@ -0,0 +1,387 @@
+/*=====================================================================
+threadsview.cpp
+------------
+
+This program is free software; you can redistribute it and/or
```

Comment:
The license header is void without a copyright line.

If you don't want to add your name, we can write something like "Copyright (C) Very Sleepy authors and contributors".